app/index.html

```diff
@@ -387,13 +387,80 @@
       <div class="Footer-column">
         <div class="Logo Logo--half"></div>
         <p class="Footer-content">
-          Bridging Portugal and Silicon Valley
+          <span>Bridging Portugal</span>
+          <span>and Silicon Valley</span>
         </p>
         <div>
           <a href="https://www.facebook.com/thewesttowest/?fref=ts" target="_blank"><img class="Footer-socialIcon" src="images/facebook-icon.png"/></a>
           <a href="https://twitter.com/TheWesttoWest" target="_blank"><img class="Footer-socialIcon" src="images/twitter-icon.png"/></a>
         </div>
       </div>
+      <div class="u-hideOnMobile">
```

Comment:
This whole section could be wrapped inside a single `u-hideOnMobile` instead of splitting it into three, couldn't it?